tests/test_TFEngine.py

```diff
@@ -66,7 +66,7 @@ def _get_tmp_file(suffix):
   return fn
 
 
-session = tf.InteractiveSession()
+tf_interactive_session = tf.InteractiveSession()
```

Comment:
Why this rename?

Reply:
`session` is used within most tests, too (`with make_scope() as session`). This hides the interactive session. This rename fixes this issue.

Reply:
Why is this a problem? When you introduce a new `session` in some of the tests, you don't need access to the interactive session.